yarn.lock

```diff
@@ -2,10 +2,19 @@
 # yarn lockfile v1
 
 
+"@agoric/babel-generator@^7.17.3":
+  version "7.17.3"
+  resolved "https://registry.yarnpkg.com/@agoric/babel-generator/-/babel-generator-7.17.3.tgz#bcd1c157e6e4daa49aa8100aada61da0a96c29ca"
+  integrity sha512-PuOwEFCy7Y9NTSGBEzF7BNG1cIaZxwFd2agPQ41bD/VOIsnq/Me4Bn4t5PGCCy74TyyQGVfh6YfF5rawizRPmQ==
+  dependencies:
+    "@babel/types" "workspace:^"
+    jsesc "condition: BABEL_8_BREAKING ? ^3.0.2 : ^2.5.1"
```

Comment:
@michaelfig is this an artifact of the publish step of `@agoric/babel-generator`? It seems like some templating didn't get applied properly

Reply:
Thank you for catching this.